packages/amplify-provider-awscloudformation/lib/push-resources.js

```diff
@@ -179,7 +179,14 @@ function storeCurrentCloudBackend(context) {
 
     const zip = archiver.create('zip', {});
     zip.pipe(output);
-    zip.directory(currentCloudBackendDir, false);
+    zip.glob(
+      '**',
+      {
+        cwd: currentCloudBackendDir,
+        ignore: ['*/*/build/**', '*/*/dist/**', '*/*/node_modules/**'],
+        dot: true,
```

Comment:
What's this for?

Reply:
This includes all the files in the directory that are not build or dist or node_modules. This decreases the size of the zip file when a project has functions with lot of dependencies